README.md

```diff
@@ -39,8 +41,9 @@
    ```sh
    npm install
    ```
+4. Update environment variables in `.env` for local development
```

Comment:
I don't think this step is necessary: all values in `.env` for the frontend should be able to be reused without issues. Did you encounter otherwise?

The one exception is if you want to use the production backend instead of local dev, but that's already mentioned in the next step.